src/global_const.js

```diff
@@ -1461,6 +1461,18 @@ module.exports.Jobs = {
         "DaBonus": 9.0,
         "TaBonus": 6.0
     },
+    "torment": {
+        "name": "トーメンター",
+        "favArm1": "dagger",
+        "favArm2": "dagger",
+        "type": "pecu",
+        "atBonus": 3000.0,
```

Comment:
atBonus is 2000 ?

Reply:
My bad read the wrong value from jp wiki. En wiki site wasn't up when I commited and pushed it.